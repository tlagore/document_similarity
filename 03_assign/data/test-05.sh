#!/usr/bin/env bash
sbt --error 'set showSuccess := false' 'run data/articles_2500.txt  " " 0.8 3 100 4 false false false'
