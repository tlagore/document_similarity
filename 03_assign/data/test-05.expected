Computing similarity with parameters: 
    Filename: data/articles_2500.txt
    Separator: [ ]
    Minimum similarity: 0.8
    Shingle size: 3
    Hash count: 100
    Band size: 4
    Compute Jaccard similarity: false
    Compute all minHashes similarity: false
    Print hash coefficients: false

Pair                     Jaccard              minHash                  lsh
--------------------------------------------------------------------------
t1088,t5015                  ---                  ---              0.92000
t1297,t4638                  ---                  ---              0.92000
t1768,t5248                  ---                  ---              0.96000
t1952,t3495                  ---                  ---              0.84000
t2023,t980                   ---                  ---              0.92000
t2535,t8642                  ---                  ---              0.96000
t2839,t9303                  ---                  ---              0.88000
t2957,t7111                  ---                  ---              0.80000
t3268,t7998                  ---                  ---              0.84000
t3466,t7563                  ---                  ---              1.00000
t3575,t8979                  ---                  ---              0.84000
t3725,t4099                  ---                  ---              0.96000
t4467,t6205                  ---                  ---              1.00000
t4530,t7907                  ---                  ---              0.92000
t5442,t906                   ---                  ---              1.00000
t5551,t7693                  ---                  ---              0.96000
t6244,t969                   ---                  ---              0.96000
t7270,t8387                  ---                  ---              0.84000
t7527,t8101                  ---                  ---              0.96000
t787,t9596                   ---                  ---              0.88000
--------------------------------------------------------------------------
